ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.js

```diff
@@ -267,6 +267,28 @@ export default {
         },
       },
     },
+    {
+      id: 'UWU 爆岩',
+      netRegex: NetRegexes.startsUsing({ id: '2B6A', source: '爆破岩石', capture: false }),
+      suppressSeconds: 1,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '爆岩',
+        },
+      },
+    },
+    {
+      id: 'UWU 流沙',
+      netRegex: NetRegexes.startsUsing({ id: '2B65', source: '泰坦', capture: false }),
+      suppressSeconds: 1,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '流沙',
+        },
+      },
+    },
```

Comment:
- `id` of a trigger should in English
- `netRegex` was for en, and `netRegexCn` was for cn.

Simply click "commit changes" button would commit my change request below.
See if this matches yours.

```suggestion
    {
      id: 'UWU Titan Bomb Burst',
      netRegex: NetRegexes.startsUsing({ id: '2B6A', source: 'bomb boulder', capture: false }),
      netRegexDe: NetRegexes.startsUsing({ id: '2B6A', source: 'Bomber-Brocken', capture: false }),
      netRegexFr: NetRegexes.startsUsing({ id: '2B6A', source: 'bombo rocher', capture: false }),
      netRegexJa: NetRegexes.startsUsing({ id: '2B6A', source: 'ボムボルダー', capture: false }),
      netRegexCn: NetRegexes.startsUsing({ id: '2B6A', source: '爆破岩石', capture: false }),
      netRegexKo: NetRegexes.startsUsing({ id: '2B6A', source: '바위폭탄', capture: false }),
      suppressSeconds: 1,
      alertText: (data, _, output) => output.text(),
      outputStrings: {
        text: {
          en: 'Bomb',
          ja: 'ボム',
          cn: '爆岩',
        },
      },
    },
    {
      id: 'UWU Titan Land Weight',
      netRegex: NetRegexes.startsUsing({ id: '2B65', source: 'Titan', capture: false }),
      netRegexDe: NetRegexes.startsUsing({ id: '2B65', source: 'Titan', capture: false }),
      netRegexFr: NetRegexes.startsUsing({ id: '2B65', source: 'Titan', capture: false }),
      netRegexJa: NetRegexes.startsUsing({ id: '2B65', source: 'タイタン', capture: false }),
      netRegexCn: NetRegexes.startsUsing({ id: '2B65', source: '泰坦', capture: false }),
      netRegexKo: NetRegexes.startsUsing({ id: '2B65', source: '타이탄', capture: false }),
      suppressSeconds: 1,
      alertText: (data, _, output) => output.text(),
      outputStrings: {
        text: {
          en: 'Weight of the Land',
          ja: '大地の重み',
          cn: '流沙',
        },
      },
    },
```

Reply:
Sorry for the review collision here!